packages/presets/src/ai/config/builder.ts

```diff
@@ -15,11 +14,10 @@ import {
   ResetIcon,
 } from '@blocksuite/blocks';
 import { assertExists } from '@blocksuite/global/utils';
-import { INLINE_ROOT_ATTR } from '@blocksuite/inline/consts';
-import type { InlineRootElement } from '@blocksuite/inline/inline-editor';
 import type { Doc } from '@blocksuite/store';
 import { Job } from '@blocksuite/store';
 
+import { getSelectedTextContent } from '../../fragments/copilot-panel/utils/selection-utils.js';
```

Comment:
The copilot-panel fragment is legacy code, we should extract the usable logic and place it in the ai folder.